packages/enhanced/src/lib/container/ContainerPlugin.ts

```diff
@@ -188,14 +188,34 @@ class ContainerPlugin {
           library,
         },
         (error: WebpackError | null | undefined) => {
-          if (error) {
-            return callback(error);
+          if (error) return callback(error);
+          if (compilation.options.optimization.runtimeChunk) {
+            // Add to single runtime chunk as well.
+            // Allows for singleton runtime graph with all needed runtime modules for federation
+            addEntryToSingleRuntimeChunk();
           }
-          callback();
         },
       );
-    });
 
+      // Function to add entry for undefined runtime
```

Comment:
Adds another entry dependency to single runtime chunk as well, this ensures all maps and container dependent functions exist in all webpack runtimes.

We never use the _partial.js, but we need to register exposed but not used by main runtime to 	`__webpack_require__.u`